webpack.config.js

```diff
@@ -54,22 +58,19 @@ const sharedConfig = {
 		new RtlCssPlugin( {
 			filename: '../css/[name]-rtl.css',
 		} ),
+		new IgnoreExtraneousStyleAssets(),
 	],
 	optimization: {
-		minimizer: [
-			new TerserPlugin( {
-				parallel: true,
-				sourceMap: false,
-				cache: true,
-				terserOptions: {
-					output: {
-						comments: /translators:/i,
-					},
-				},
-				extractComments: false,
-			} ),
```

Comment:
Out of curiosity: why are we no longer overriding the `TerserPlugin` config (or why have we done so before at all)?

Reply:
The config we're importing from `@wordpress/scripts` already [configures](https://github.com/WordPress/gutenberg/blob/trunk/packages/scripts/config/webpack.config.js#L124-L141) the `TerserPlugin` the same way we do (with the additional config of not mangling translations).

As for why we were overriding it, we actually were not. The `TerserPlugin` config was only "recently" added to the default Webpack config (see https://github.com/WordPress/gutenberg/pull/22990) (relative to when this plugin config was first added, that is).

Reply:
Thanks for the explanation 🙏